modules/api/src/test/java/org/apache/ignite/table/QualifiedNameTest.java

```diff
@@ -273,9 +270,4 @@ public void malformedCanonicalNames(String source) {
                 containsString("Malformed identifier [identifier=" + source)
         )));
     }
-
-    // TODO https://issues.apache.org/jira/browse/IGNITE-24021 Move to IgniteNameUtils
-    private static String canonicalName(String schemaName, String objectName) {
-        return schemaName == null ? quoteIfNeeded(objectName) : quoteIfNeeded(schemaName) + '.' + quoteIfNeeded(objectName);
```

Comment:
There is no reason to allow `null` for `schemaName`. We using `quoteIfNeeded` instead.

Alternatively we can change method signature to 
```
canonicalName(String ... names)
```
and replace quoteIfNeeded(name) with canonicalName(name) everywhere.... thus making `quoteIfNeeded` private :thinking: